src/pages/about/about.js

```diff
@@ -82,6 +97,80 @@ export default function About() {
           </div>
         </div>
       </section>
+
+      <section className="section-four">
```

Comment:
Try to use a more descriptive className. Not only does section four not tell us what this section should contain, it can also be a problem if the className section-four is reused in a parent scope or child scope. That can lead to css being overwritten in unintended ways. This can also be avoided by using `id={}` instead of className for things that are intended to be unique and not repeated.

Reply:
The above comment applies to a number of the generic className tags below. At the very least they should be contact-form-background, so that they are more unique and identified as belonging to the contact form.